src/main/java/com/cisco/trex/stateful/model/stats/PortStatsData.java

```diff
@@ -0,0 +1,182 @@
+package com.cisco.trex.stateful.model.stats;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class PortStatsData {
+
+    @JsonProperty("ignore_bytes")
+    public Integer ignore_bytes;
```

Comment:
naming with java convention, same comments for all below fields

Reply:
ok, I will change it to camel case.